src/main/resources/uischema/addAttachmentToIssueId.json

```diff
@@ -0,0 +1,80 @@
+{
+  "connectorName": "jira",
+  "operationName": "addAttachmentToIssueId",
+  "title": "Add Attachment to Issue",
+  "help": "Attach a file to a Jira issue using its ID or key. Ensure the payload is a multipart/form-data request containing the file.",
+  "elements": [
+    {
+      "type": "attributeGroup",
+      "value": {
+        "groupName": "General",
+        "elements": [
+          {
+            "type": "attribute",
+            "value": {
+              "name": "configRef",
+              "displayName": "Jira Connection",
+              "inputType": "connection",
+              "allowedConnectionTypes": [
+                "JIRA"
+              ],
+              "defaultType": "connection.JIRA",
+              "defaultValue": "",
+              "required": "true",
+              "helpTip": "Connection configuration used to access the Jira server."
+            }
+          },
+          {
+            "type": "attributeGroup",
+            "value": {
+              "groupName": "Attachment Input",
+              "elements": [
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "issueIdOrKey",
+                    "displayName": "Issue ID or Key",
+                    "inputType": "stringOrExpression",
+                    "defaultValue": "",
+                    "required": "true",
+                    "helpTip": "The Jira issue ID or key to which the attachment will be added."
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "type": "attributeGroup",
```

Comment:
Shall we call this Output and use the same format across the connectors similar to https://github.com/wso2-extensions/esb-connector-file/blob/master/src/main/resources/uischema/read.json#L241-L270? 

Lets change in other places as well.